docs.kosli.com/content/getting_started/attestations.md

```diff
@@ -5,9 +5,11 @@ weight: 270
 ---
 # Part 7: Attestations
 
-Attestations are how you record the facts your care about in your software supply chain. They are the evidence that you have performed certain activities, such as running tests, security scans, or ensuring that a certain requirement is met.
+Attestations are how you record the facts your care about in your software supply chain. 
```

Comment:
replace `the facts your care about` with `the facts you care about`

Reply:
Good spot.